src/content/getting-started/architecture/_index.en.md

```diff
@@ -39,5 +39,8 @@ be created manually or via the `subctl export` command.
 
   * When a Service is exported, it then becomes accessible as `<service>.<ns>.svc.clusterset.local`.
 
+  * For Headless Services, individual pods can be accessed as `<pod-name>.<cluster-id>.<svc-name>.<ns>.svc.clusterset.local`
```

Comment:
```suggestion
  * For Headless Services, individual Pods can be accessed as `<pod-name>.<cluster-id>.<svc-name>.<ns>.svc.clusterset.local`
```

If we're going to capitalize "Headless Services" right before it, it seems like we should capitalize "Pods".